CHANGELOG.md

```diff
@@ -44,37 +47,23 @@ This name should be decided amongst the team before the release.
 * [#533](https://github.com/tweag/topiary/pull/533) Update tree-sitter-ocaml to 0.20.3
 * [#535](https://github.com/tweag/topiary/pull/535) Improved error message when idempotency fails due to invalid output in the first pass.
 
-## [0.2.3] - 2023-06-20
```

Comment:
It seems a shame to lose the release date.

Reply:
I agree with this. While the GitHub release do have a date associated with them, I would like the repo to be able to stand on it's own.

Reply:
We can put the dates back in, they'll just be slightly redundant on the GitHub release page.

Reply:
Either way, I'm happy.

Reply:
Fixed.